cmd/loadbalancer/loadbalancer.go

```diff
@@ -21,26 +21,50 @@ var LoadBalancerCmd = &cobra.Command{
 }
 
 func init() {
+	lbBackendExample := `Define backend configurations for the load balancer.
+Use this format:
+--backend "ip=10.0.0.1|protocol=http|source-port=80|target-port=8080|health-check-port=8080"
```

Comment:
are these the minimal required fields to be set by the user?
or there are some that are optional? 

I think we should generally improve the way we say a field is optional or not